Leaf API.postman_collection.json

```diff
@@ -2372,587 +2372,775 @@
-							},
-							"response": []
-						},
-						{
-							"name": "Get intersection of fields",
-							"event": [
 								{
-									"listen": "test",
-									"script": {
-										"exec": [
-											"pm.test(\"Successful POST request\", function () {",
-											"    pm.expect(pm.response.code).to.be.oneOf([200]);",
-											"});"
+									"name": "Get a boundary of a field",
+									"request": {
+										"auth": {
+											"type": "noauth"
+										},
+										"method": "GET",
+										"header": [
+											{
+												"key": "Content-Type",
+												"value": "application/json",
+												"type": "text"
+											},
+											{
+												"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
+												"key": "Authorization",
+												"value": "Bearer {{leaf_token}}",
+												"type": "text"
+											}
 										],
-										"type": "text/javascript"
-									}
-								}
-							],
-							"request": {
-								"method": "POST",
-								"header": [
-									{
-										"key": "Authorization",
-										"value": "Bearer {{leaf_token}}"
+										"url": {
+											"raw": "https://{{leaf_api_url}}/services/fields/api/users/{{leaf_user_id}}/fields/{{field_id}}/boundaries/{{boundary_id}}",
+											"protocol": "https",
+											"host": [
+												"{{leaf_api_url}}"
+											],
+											"path": [
+												"services",
+												"fields",
+												"api",
+												"users",
+												"{{leaf_user_id}}",
+												"fields",
+												"{{field_id}}",
+												"boundaries",
+												"{{boundary_id}}"
+											]
+										}
 									},
-									{
-										"key": "Content-Type",
-										"value": "application/json"
-									}
-								],
-								"body": {
-									"mode": "raw",
-									"raw": "[\"{{field_id}}\", \"{{field_id}}\"]\n\n// [\"field1 id\", \"field2 id\", \"fieldN id\"]\n\n"
-								},
-								"url": {
-									"raw": "https://{{leaf_api_url}}/services/fields/api/users/{{leaf_user_id}}/fields/intersect",
-									"protocol": "https",
-									"host": [
-										"{{leaf_api_url}}"
-									],
-									"path": [
-										"services",
-										"fields",
-										"api",
-										"users",
-										"{{leaf_user_id}}",
-										"fields",
-										"intersect"
-									]
+									"response": []
 								},
-								"description": "If there are fields that intersect with the given geometry, they will be returned."
-							},
-							"response": []
-						},
-						{
-							"name": "Delete a field",
-							"request": {
-								"method": "DELETE",
-								"header": [
-									{
-										"key": "Authorization",
-										"value": "Bearer {{leaf_token}}",
-										"type": "text"
+								{
+									"name": "Add active boundary of a field",
```

Comment:
```suggestion
									"name": "Add active boundary of a field",
```

Reply:
I think you forgot to edit it to the proposed phrasing hahaha

Reply:
get*

Reply:
Nice, fixed in 684f6338a1c2551a59ec2a9dbb2ac551bf1a3351